services/mailbox.org.lua

```diff
@@ -0,0 +1,24 @@
+local M       = {}
+
+M.name        = "Mailbox.org"
+M.description = "Mailbox.org is an email hosting provider."
+M.category    = "email"
+M.fields      = {}
+M.records     = function(_)
+    return {
+        -- Receiving MX servers, required for receiving.
+        mx("", "mxext1.mailbox.org.", 10)
```

Comment:
This is just a convention, we do prefer to use `""` in record names because the record name is relative to the zone name. The `_a` global variable is used usually in the _content_ part of the record.

In the **_example.com.lua** context the`""` name is actually `"example.com"` which equals to `_a` global variable. 😄